packages/vue/src/components/confirm-sign-up.vue

```diff
@@ -42,6 +46,8 @@ const onInput = (e: Event): void => {
 };
 
 const onConfirmSignUpSubmit = (e: Event): void => {
+  // TODO(BREAKING): remove unused emit
+  // istanbul ignore next
```

Comment:
As with the previous PR, we are intentionally ignoring coverage for `emit`, because this is undocumented and unused API. This `emit` has been marked deprecated above.